jina/docker/hubio.py

```diff
@@ -344,8 +344,13 @@ def build(self) -> Dict:
                     p_names = []
                     try:
                         is_build_success = False
-                        p_names = self._test_build(image)
-                        is_build_success = True
+                        p_names, failed_test_levels = self._test_build(image)
+                        if BuildTestLevel.POD_DOCKER or BuildTestLevel.FLOW in failed_test_levels:
+                            is_build_success = False
+                            self.logger.error(f'build unsuccessful, failed at {str(failed_test_levels).strip('[]')} level')
+                        else:
+                            is_build_success = True
+                            self.logger.warning(f'test failed at : {str(failed_test_levels).strip('[]')} levels')
```

Comment:
Clarify the message maybe? 'Build successful. Tests failed at this level ... (This could be do to the fact that `executor` has non-installed external dependencies)